_data/finance.yml

```diff
@@ -13,318 +13,290 @@ websites:
-      tfa: No
+      bch: No
 
     - name: Noddle
       url: https://www.noddle.co.uk
       twitter: useyournoddle
       facebook: noodleuk
       img: noddle.png
-      tfa: No
+      bch: No
 
     - name: Northwestern Mutual
       url: https://www.northwesternmutual.com/
       facebook: northwesternmutual
       img: northwesternmutual.png
-      tfa: No
+      bch: No
 
     - name: Paychex
       url: https://www.paychex.com
       img: paychex.png
-      tfa: Yes
-      sms: Yes
-      phone: Yes
+      bch: No
 
     - name: Paycor
       url: https://www.paycor.com
       img: paycor.png
-      tfa: Yes
-      sms: Yes
-      email: Yes
-      phone: Yes
-      doc: https://www.paycor.com/security/how-to-keep-your-account-safe
+      bch: No
 
     - name: Paytrust
       url: http://paytrust.quicken.com
       img: paytrust.png
-      tfa: Yes
-      phone: Yes
-
+      bch: No
+      
     - name: Pocketsmith
       url: https://www.pocketsmith.com
       img: pocketsmith.png
-      tfa: Yes
-      software: Yes
-      doc: https://www.pocketsmith.com/blog/2014/07/18/2fa-orginal-merchant-and-other-updates
+      bch: No
 
     - name: Popmoney
       url: https://www.popmoney.com
       twitter: popmoney
       facebook: popmoney
       img: popmoney.png
-      tfa: No
+      bch: No
 
     - name: President's Choice Financial
       url: https://www.pcfinancial.ca
       twitter: PCFinancial
       facebook: PCFinancial
       img: presidentschoice.png
-      tfa: No
+      bch: No
 
     - name: Progressive
       url: https://www.progressive.com
       twitter: progressive
       facebook: progressive
       img: progressive.png
-      tfa: No
+      bch: No
 
     - name: Quickbooks Online
       url: https://quickbooks.intuit.com/
       img: quickbooks.png
-      tfa: Yes
-      sms: Yes
-      phone: Yes
-      software: Yes
-      doc: https://community.intuit.com/articles/1164912-multi-factor-authentication-faq
+      bch: No
 
     - name: QuickFile
       url: https://www.quickfile.co.uk
       img: quickfile.png
-      tfa: Yes
-      software: Yes
-      doc: https://community.quickfile.co.uk/t/2-factor-authentication/8892
+      bch: No
 
     - name: Quizzle
       url: https://www.quizzle.com
       img: quizzle.png
-      tfa: No
       twitter: Quizzle
+      bch: No
 
     - name: Shoeboxed
       url: https://www.shoeboxed.com/
       twitter: Shoeboxed
       facebook: shoeboxed
       img: shoeboxed.png
-      tfa: No
+      bch: No
 
     - name: SimpleTax
       url: https://simpletax.ca/
       img: simpletax.png
-      tfa: Yes
-      software: Yes
-      doc: https://help.simpletax.ca/questions/two-factor-authentication
+      bch: No
 
     - name: Taxact
       url: https://www.taxact.com
       img: taxact.png
-      tfa: Yes
-      email: Yes
-      sms: Yes
-      doc: https://www.taxact.com/support/24138/2016/how-is-a-verification-code-used
+      bch: No
+
+    - name: Tradingview
```

Comment:
This seems like a cryptoservice, not a finance one per se. Also, I think they do accept BCH and other cryptos, but please verify when moving

Reply:
Might actually be a cryptoexchange.